mailpoet/assets/js/src/email-editor/engine/components/styles-sidebar/dimensions-panel.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  __experimentalSpacingSizesControl as SpacingSizesControl,
+  useSetting,
+} from '@wordpress/block-editor';
+import {
+  __experimentalToolsPanel as ToolsPanel,
+  __experimentalToolsPanelItem as ToolsPanelItem,
+  __experimentalUseCustomUnits as useCustomUnits,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+export function DimensionsPanel() {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const availableUnits: string[] = useSetting('spacing.units');
+  const units = useCustomUnits({
+    availableUnits,
+  });
+
+  // Padding
+  const paddingValues = {
+    top: '20px',
+    right: '20px',
+    bottom: '20px',
+    left: '20px',
+  };
+  const resetPadding = () => {};
+  const setPaddingValues = () => {};
+
+  // Block spacing
+  const blockGapValue = '16px';
+  const resetBlockGap = () => {};
+  const setBlockGapValue = () => {};
+  const resetAll = () => {};
+
+  return (
+    <ToolsPanel label={__('Dimensions', 'mailpoet')} resetAll={resetAll}>
+      <ToolsPanelItem
+        isShownByDefault
+        hasValue={() => true}
+        label={__('Padding')}
+        onDeselect={() => resetPadding()}
+        className="tools-panel-item-spacing"
+      >
+        <SpacingSizesControl
+          allowReset
+          values={paddingValues}
+          onChange={setPaddingValues}
+          label={__('Padding', 'mailpoet')}
```

Comment:
I think we should provide a comment for translators here. OR, I see a couple of lines above where `__('Padding')` is used without MailPoet's namespace. Hmm 'Padding' is commonly used in the block editor and we load the module from the WP so we should be able also to load it's translations (maybe we already do I'm not 100% sure) I guess that for `Padding` and also for `Block spacing` we could skip adding `mailpoet` namespace.

Reply:
Using those translations without a namespace makes sense to me here. I will update it.